app/resources/v1/article_resource.rb

```diff
@@ -30,8 +30,15 @@ def fetchable_fields
     super - [:cover_photo]
   end
 
-  def self.creatable_fields(_context)
-    %i[title content publicly_visible group cover_photo]
+  def self.creatable_fields(context)
+    attributes = %i[title content publicly_visible group cover_photo]
+
+    attributes += [:pinned] if update_permission?(context)
```

Comment:
great! Could you write a little spec for this? See for example `/spec/resources/v1/user_resource_spec.rb`